### DISCUSSION TITLE
If you are on Mac, instead of the CFLAGS, you can update your xCode command line tools

### DISCUSSION THREAD

**Discussion by zenware** [pinax/pinax-stripe#655] [Show and tell]
This is accomplished by running the command `xcode-select --install`, if the permissions or prefixes have been messed up at some point, this should handle it, and I think it's easier to follow than the compiler and linker flags currently in the readme.

**Comment by ossanna16** [comment]
@paltman Thoughts on this approach? Thank you for the idea @zenware :)

**Comment by paltman** [comment]
I am missing the context for what the title of this issue is referring to.  Please advise.

**Comment by brosner** [comment]
I am assuming this is referencing the development section of the README.

**Comment by paltman** [comment]
Ah, yes!  @zenware please submit a PR with this.  Anything that makes things more clear I'm +1  for.

**Comment by zenware** [comment]
Wow, I'm clearly an exceptionally quick responder. smh
I'm pretty sure it was the dev section of the readme, I could've made the title or description more clear, and I will actually have to re-investigate this quickly... Hopefully, while I'm waiting with nothing to do at the airport tonight. I'll confirm that it still works that way and then if so I'll PR the readme and if not I'll close the issue or turn that section of the readme into a shell script or something.
